config/parse.py

```diff
@@ -313,9 +313,9 @@ def transform_for_keys(element, keys, transform_func):
         )
 
         pmem = util.chain(self.pmem, {
```

Comment:
There are ways to keep support for `"frequency"`, which defaults to half the data rate, if you want.

Reply:
I think these changes I am about to push will solve this. In my code, frequency is half the data rate. To the user, frequency or data_rate can be used in the config, but internally frequency sets the data_rate to frequency and then halves itself before being passed on to the rest of the config script.

I am not sure if there's a cleaner way to do this, but it allows for support of both.

Reply:
Oh, I see, you've got an if-clause up the file a bit. Yea, it's kinda clunky but it works...